model/src/test/kotlin/config/AnalyzerConfigurationTest.kt

```diff
@@ -138,7 +138,7 @@ class AnalyzerConfigurationTest : WordSpec({
                 packageManagers = mapOf(
                     "Gradle" to PackageManagerConfiguration(
                         mustRunAfter = listOf("Npm"),
-                        options = mapOf("gradleVersion" to "7.6.1")
+                        options = mapOf("gradleVersion" to "7.6.1", "javaHome" to "/path/to/java/home")
```

Comment:
This should be an option of "GradleInspector", not "Gradle" (same below). Adding it here as a "Gradle" option could confuse someone.

Reply:
Agreed. Actually, I was just adding it here "for fun" to further show that replaced options do not discard other existing options. But for simplicity, I'm inclined to just drop this change. Any objections?

Reply:
Scratch that, I just made this particular test use "GradleInspector", which is what you suggested I believe, I just misread it initially.